### PR TITLE
Disable autocomplete for campaign name.

### LINKED ISSUE #135: Turn off autocomplete in "campaign name" field.
```suggestion
                <InterestFormTextInput name="campaignName" label="Campaign Name" placeholder="Give your campaign a name." />
```

One more thing, when I tested it, it was giving me name (siva, bharath, etc) autocomplete suggestions. May want to turn that off.

_Originally posted by @sivayogasubramanian in https://github.com/Giving-Coupons/giving-coupons/pull/133#discussion_r994787828_

### PR DESCRIPTION
**Pull request by bharathcs**
Fixes #135.

### CODE REVIEW

File: frontend/components/interests/form/InterestFormTextInput.tsx
Comment (sivayogasubramanian) [inline frontend/components/interests/form/InterestFormTextInput.tsx:30]: Why is there an extra `,`?

File: frontend/components/interests/form/InterestFormTextInput.tsx
Comment (sivayogasubramanian) [inline frontend/components/interests/form/InterestFormTextInput.tsx:55]: Howcome the other instances of `InterestFormTextInput` do not have the `disableAutocomplete` and the autocomplete is off? Like the Description and Length of Campaign.

**Review by sivayogasubramanian — APPROVED** [review]
LGTM! Can merge once the comments are resolved.

File: frontend/components/interests/form/InterestFormTextInput.tsx
Comment (bharathcs) [reply frontend/components/interests/form/InterestFormTextInput.tsx:30]: What do you mean?  The last item in an opened braces will always have a trailing comma according to Prettier, right?
```
{
  ....
  x,
}
```

File: frontend/components/interests/form/InterestFormTextInput.tsx
Comment (bharathcs) [reply frontend/components/interests/form/InterestFormTextInput.tsx:55]: Because your browser was unable to autocomplete description and length of campaign, so didn't bother suggesting any options for you.

File: frontend/components/interests/form/InterestFormTextInput.tsx
Comment (bharathcs) [inline frontend/components/interests/form/InterestFormTextInput.tsx:12]: @zognin just realised you are working on these files as well for #130. Lets merge yours first. 

⚠️  Do not resolve this conversation, do not merge this PR until the above is done.

File: frontend/components/interests/form/InterestFormTextInput.tsx
Comment (sivayogasubramanian) [reply frontend/components/interests/form/InterestFormTextInput.tsx:30]: I see

File: frontend/components/interests/form/InterestFormTextInput.tsx
Comment (sivayogasubramanian) [reply frontend/components/interests/form/InterestFormTextInput.tsx:55]: Okay

**Pull request merged**
